tests/out/trips.txt

```diff
@@ -0,0 +1,627 @@
+route_id,service_id,trip_id,trip_headsign,shape_id
+19914,WE,trp_19914,Dummy Trip,shp_19914
```

Comment:
@mehdisadeghi would it be possible to have the relation "name" as the headsign? This can obviously be handled in another PR but I just wanted to ask it here since the context is pretty useful ;)

Reply:
@carocad hmm. Yes of course, however I wanted to make it obvious that this is a dummy trip. Perhaps a combination of relation + dummy will do it.

Reply:
yeah I understand that differentiate those is important. I was thinking more in terms of the front-end. There we need to display something to the user such that it is clear which route/trip are we referring to. Obviously showing `Dummy trip` to a user would not be very useful. I think that just as you said, having a combination of real + dummy would be ideal. That way we can strip the dummy part at runtime and display only a nice headsign.